### PR TITLE
Payment details

### PR DESCRIPTION
**Pull request by tobyprivett**
https://eaflood.atlassian.net/browse/RUBY-1548

This PR styles the Back Office Payments details pages

### CODE REVIEW

File: app/views/finance_details/show.html.erb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 </div>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <%= render "shared/company_details_panel", registration: @resource %>
 
-    <% if @resource.pending_payment? %>
+    <% if !@resource.pending_payment? %>
```
Comment (irisfaraway) [inline app/views/finance_details/show.html.erb:17]: What's the reason for this change?

File: app/views/positive_charge_adjust_forms/new.html.erb [hunk 2]
```diff
@@ -1,61 +1,70 @@
...
-          <span class="form-hint">
-            <%= t(".fields.description.hint") %>
-          </span>
-        <% end %>
+            <span class="govuk-hint">
+              <%= t(".fields.description.hint") %>
+            </span>
+
+            <div id="with-hint-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
+              You can enter up to 200 characters
```
Comment (irisfaraway) [inline app/views/positive_charge_adjust_forms/new.html.erb:55]: Should this text be in an I18n file instead?

**Review by irisfaraway — COMMENTED** [review]
Looks good for the most part – just had some minor questions.

File: app/views/finance_details/show.html.erb (hunk 1 above)
Comment (tobyprivett) [reply app/views/finance_details/show.html.erb:17]: That looks like my error

File: app/views/positive_charge_adjust_forms/new.html.erb (hunk 2 above)
Comment (tobyprivett) [reply app/views/positive_charge_adjust_forms/new.html.erb]: That text shouldn't be there. Well spotted!. The <div> is there to help the 'You have X characters remaining' text.

**Review by irisfaraway — DISMISSED** [review]

**Review by irisfaraway — APPROVED** [review]
